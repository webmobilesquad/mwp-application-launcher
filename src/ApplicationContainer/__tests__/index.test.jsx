import render from 'preact-render-spy';
import { ApplicationContainer } from '../index';

describe('<ApplicationContainer />', () => {
  test('<ApplicationrContainer /> matches snapshot', () => {
    expect(render(<ApplicationContainer />)).toMatchSnapshot();
  });
});
