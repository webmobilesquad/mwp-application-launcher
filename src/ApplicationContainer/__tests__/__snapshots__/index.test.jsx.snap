// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApplicationContainer /> <ApplicationrContainer /> matches snapshot 1`] = `
preact-render-spy (1 nodes)
-------
<div class="ApplicationContainer"></div>

`;
